core/src/components/slides/slides.tsx

```diff
@@ -521,6 +521,9 @@ export class Slides implements ComponentInterface {
         <div class="swiper-wrapper">
           <slot></slot>
         </div>
+        
+        <slot name="container"></slot>
```

Comment:
In addition to adding the slot, we need to update the component file so this is automatically listed on our docs site. See https://github.com/ionic-team/ionic-framework/blob/master/core/src/components/item/item.tsx#L12-L14 for an example.

Reply:
Done